proto/cosmos/crisis/v1/tx.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+package cosmos.crisis.v1;
+
+import "cosmos/msg/v1/msg.proto";
+
+// Msg defines the crisis Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // AuthorizeCircuitBreaker allows a super-admin to grant (or revoke) another
+  // account's circuit breaker permissions.
+  rpc AuthorizeCircuitBreaker(MsgAuthorizeCircuitBreaker) returns (MsgAuthorizeCircuitBreakerResponse);
+
```

Comment:
I think I concur with @tomtau to have a simpler design in x/crisis, i.e. one single circuit breaker address that can trip all Msgs. We don't need Authorize anymore in this case, just Trip and Reset.

Then, optionally, this single address can create authz grants to other addresses.

```proto
message CircuitBreakerAuthorization {
  // msg_type_urls are Msgs that the grantee can trip. If left empty, then
  // the grantee can trip **ALL** Msgs.
  repeated string msg_type_urls = 2;
}
```

And the only difference is that grantees would need to call `MsgExec{MsgTrip}` instead of `MsgTrip` directly with the current design.